{{ cookiecutter.repo_name }}/requirements/base.txt

```diff
@@ -0,0 +1,255 @@
+#
```

Comment:
IMO we should leave this file out with the understanding that when someone generates a repo one of the first orders of business is to `make pip-compile`. They are going to need to do that anyway.